src/components/scenes/Staking/StakeOptionsScene.tsx

```diff
@@ -117,7 +115,7 @@ const StakeOptionsSceneComponent = (props: Props) => {
       {({ undoInsetStyle, insetStyle }) => (
         <SceneContainer undoBottom undoInsetStyle={undoInsetStyle}>
           <FlatList
-            data={stakePolicies}
+            data={Object.values(stakePolicies)}
```

Comment:
I think this will cause a full list refresh on every render. You need to do `const policyList = React.useMemo(()=> Object.values(stakePolicies), [stakePolicies])` to cache the value, so the list only gets new `data` if `stakePolicies` actually changes.